PhpAmqpLib/Channel/AbstractChannel.php

```diff
@@ -218,7 +218,7 @@ public function next_frame($timeout = 0)
     }
 
     /**
-     * @param $method_sig
+     * @param string $method_sig
```

Comment:
A `$method_sig` argument is an array in these methods:

`AbstractChannel#send_method_frame()`
`AbstractConnection#prepare_channel_method_frame()`
`AbstractConnection#send_channel_method_frame()`

A `$method_sig` argument is a string in these methods:

`AbstractChannel#dispatch()`

I've changed the appropriate doc blocks to reflect these types.